### PR TITLE
Merge CI updates

### PR DESCRIPTION
**Pull request by buggtb**
Ensures a tag is created correctly. Artifacts deployed to S3

### CODE REVIEW

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -179,13 +179,16 @@ jobs:
...
     runs-on: ubuntu-latest
     steps:
       - name: Trigger Job
         run: |
           curl -X POST \
                --fail \
                -F token=${{ secrets.GITLABTOKEN }} \
-               -F ref=main \
+               -F ref="feat/env-stacks" \
+               -F "variables[STACK_NAME]=dev" \
```
Comment (michaelfedell) [inline .github/workflows/dev.yml:193]: Have you tested this?

File: .github/workflows/features.yml [hunk 2]
```diff
@@ -21,10 +21,37 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: paulhatch/semantic-version@v5.0.0-alpha2
```
Comment (michaelfedell) [inline .github/workflows/features.yml:26]: looks cool

**Review by michaelfedell — APPROVED** [review]
Changes seem pretty straightforward; lgtm

File: .github/workflows/dev.yml (hunk 1 above)
Comment (buggtb) [reply .github/workflows/dev.yml:193]: yup

**Pull request merged**